.buildkite/autogenerate_pipeline.py

```diff
@@ -363,5 +392,13 @@ def generate_pipeline(config_file):
         help="The path to the JSON file containing the test" " description for the CI.",
         default=f"{PARENT_DIR}/test_description.json",
     )
+    parser.add_argument(
+        "-p",
+        "--platform-allowlist",
+        metavar="PLATFORM_DOT_FILE",
+        help="The path to the dot file containing the allowed platforms for the CI.",
+        default=f"{REPO_ROOT_DIR}/.platform",
```

Comment:
What about using something similar of what we do for `coverage_config_{machine}.json` ?

See https://github.com/rust-vmm/rust-vmm-ci/blob/cec97d9e6c578949d40226b8b85fa4ad3142e31f/integration_tests/test_coverage.py#L16